overlord/ifacestate/helpers.go

```diff
@@ -429,6 +442,22 @@ func (m *InterfaceManager) autoConnect(task *state.Task, snapName string, blackl
 	return affectedSnapNames, nil
 }
 
+func getAutoconnectDisabled(st *state.State) (map[string]bool, error) {
+	var conns map[string]bool
+	err := st.Get("autoconnect-disabled", &conns)
+	if err != nil {
+		if err != state.ErrNoState {
+			return nil, err
+		}
+		conns = make(map[string]bool)
+	}
+	return conns, nil
```

Comment:
I think it'd do:
```
if err != nil && err != state.ErrNoState {
	return nil, err
}
if conns == nil {
	conns = make(map[string]bool)
}
```
Just in case "autoconnect-disabled" is `null` in state.json (is it even possible?).

Reply:
No, that's not possible, it's  a map that we create and maintain.